src/models/tools/geometry/geometry-content.ts

```diff
@@ -46,14 +46,15 @@ export interface IAxesParams {
   yMax: number;
 }
 
-export function defaultGeometryContent(overrides?: JXGProperties): GeometryContentModelType {
-  const { title, ...boardProps } = overrides || {};
+// This is only used directly by tests
+export function defaultGeometryContent(options?: IDefaultContentOptions): GeometryContentModelType {
```

Comment:
I should have thought of this when we were talking about it before, but if it's only used by the test code, could it be moved into the test code?

Reply:
It is tricky. `defaultGeometryContent` is only used **directly** by the test code. But it is used indirectly by the runtime code via the `defaultContent` function that is registered with `registerToolContentInfo`.

In both cases it seems necessary to automatically add the title change and default board change. So it seems worthwhile to keep this function in `geometry-content.ts` so the runtime code is exercised by more tests.